packages/next-codemod/transforms/lib/async-request-api/next-async-dynamic-prop.ts

```diff
@@ -175,7 +176,55 @@ function applyUseAndRenameAccessedProp(
   return modified
 }
 
-const MATCHED_FILE_PATTERNS = /([\\/]|^)(page|layout|route)\.(t|j)sx?$/
+function commentOnMatchedReExports(
+  root: Collection<any>,
+  j: API['jscodeshift']
+) {
+  let modified = false
+  root.find(j.ExportNamedDeclaration).forEach((path) => {
+    if (j.ExportSpecifier.check(path.value.specifiers[0])) {
+      const specifiers = path.value.specifiers
+      for (const specifier of specifiers) {
+        if (
+          j.ExportSpecifier.check(specifier) &&
+          // Find matched named exports and default export
+          (TARGET_NAMED_EXPORTS.has(specifier.exported.name) ||
+            specifier.exported.name === 'default')
+        ) {
+          if (j.Literal.check(path.value.source)) {
+            const localName = specifier.local.name
+            insertCommentOnce(
+              specifier,
+              j,
+              ` Next.js Dynamic Async API Codemod: \`${localName}\` export is re-exported. Check if this component uses \`params\` or \`searchParams\``
+            )
+            modified = true
```

Comment:
Shouldn't this only be flagged as `modified` if `insertCommentOnce` actually inserted a comment?